applications/Societal.md

```diff
@@ -0,0 +1,214 @@
+
+Kim Fournier is an advocate for blockchain marketing and is passionate about the intersection of decentralization and the future of work. She excels at crafting strong brand narratives to help organizations scale, and has led successful product releases, working with some of the world's top companies and fastest-growing startups both in and outside the blockchain space, including five Layer1 chains. She is an active member of the NFT community, as well as CryptoFemmes and Women in Blockchain.
+
+Max Kudinov is the CEO of Magic Powered, a software development firm building multiple web3 projects. Max has a passion for governance projects and was the Project Lead for the development of [Astro DAO](https://astrodao.com/), an all-in-one DAO creation and management platform built on Near. From this experience, Max has excellent understanding of Rust and how he would like to improve upon the Astro DAO project. Max has highly experienced web3 and rust developers on his team, ready to start the development of this project. 
+
+
+### Team Code Repos
+
+Org:
+- https://github.com/sctllabs
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+Team:
+- https://github.com/gfox1
+- https://github.com/near-daos
+  - https://github.com/near-daos/astro-ui
+  - https://github.com/near-daos/astro-api-gateway
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/graemefox/
+- https://www.linkedin.com/in/kimjfournier/
+- https://www.linkedin.com/in/tngell/
+- https://www.linkedin.com/in/max-kudinov-395595152/
+
+## Development Status :open_book:
+
+Before applying for the Web3 Foundation Grant, the Societal team conducted the following research:
+
+- Reviewed the list of ideas that the Web3 Foundation could be interested in Funding [here](https://github.com/w3f/Grants-Program/blob/master/docs/polkadot_stack.md), our focus was on governance and treasury management UI
+- Reviewed the list of improvement proposals and the one that stood out the most for our current roadmap was [XCM library & tools](https://github.com/w3f/Grants-Program/blob/master/rfps/xcm-tool.md#xcm-library--tools)
```

Comment:
Applications don't need to strictly follow a request for proposals. The RFPs are there to encourage teams to work in this area, but it's no issue if the application doesn't find into any of them.

Reply:
Yes, I understand. I just wanted to show here that I have done research on the RFPs and am trying to build something that is wanted in the ecosystem.